fetch.bs

```diff
@@ -4387,12 +4387,18 @@ steps:
 
       <p class=note>This covers the case of <var>response</var> being a <a>network error</a>.
 
+     <li><p>Let <var>responseStatus</var> be 0 if <var>fetchParams</var>'s
+     <a for="fetch params">request</a>'s <a for=request>mode</a> is "<code>navigate</code>" and
+     <var>response</var>'s <a for=response>URL</a>'s <a for=url>origin</a> is not
+     <a>same origin</a> with <var>request</var>'s <a for=request>origin</a>, and
+     <var>response</var>'s <a for=response>status</a> otherwise.
```

Comment:
The style we try to follow (see Infra) is like this:

> origin; otherwise _response_'s status.

Reply:
Updated